components/single_image_view/single_image_view.test.jsx

```diff
@@ -100,35 +100,6 @@ describe('components/SingleImageView', () => {
         expect(wrapper.state('showPreviewModal')).toEqual(false);
     });
 
-    test('should match dimensions on computeImageDimensions', () => {
```

Comment:
Using a utils functions which is used in most places now. It covers more corner cases and is tested there for this scenario